site/pages/meet-our-team/team-slice/job-advert/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Link from '../../../../components/link';
+import styles from './style.css';
+
+const JobAdvert = () => (
+  <Link to="joinUs" className={styles.jobAdvert}>
+    <div className={styles.question}>Are you a potential Badger?</div>
+    <div className={styles.hiring}>We're hiring</div>
```

Comment:
It's annoying but you might want to use the escaped version of `'` as this also breaks some syntax highlighting :(